website/project/model.py

```diff
@@ -3729,3 +3810,125 @@ def _on_reject(self, user, token):
             },
             auth=Auth(user),
         )
+
+
+class DraftRegistrationApproval(Sanction):
+
+    mode = Sanction.ANY
```

Comment:
If I read this correctly, the draft registration will be formalized if any single person approves it. Is that the desired behavior?

(particularly relevant given that prereg challenge offers monetary incentives to be the author that pushes the button)

See L3291 for `_on_approve` logic.